store.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+</head>
+
+<body>
+
+
+    <nav>
+        <a href="index.html"><img class="logo" src="img/karma-logo.svg" alt="Meet Karma Logo"></a>
+        <ul class="nav__links">
+            <li><a href="index.html">Meet Karma</a></li>
+            <li><a href="#">How It Works</a></li>
+            <li><a href="#" class="nav__link_on">Store</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Help</a></li>
+            <li><a href="#">Login</a></li>
+        </ul>
+    </nav>
+
+
+    <main class="main">
+        <div class="form">
+            <h1>Order your Karma wifi device today!</h1>
+            <form>
+                <fieldset>
+                    <div class="field-flex">
+                        <div class="field-fname">
+
+                            <label for="fname">First name *</label><br>
+                            <input type="text" id="fname" name="fname" required>
+                        </div>
+
+                        <div class="field-lname">
+                            <label for="lname">Last name *</label><br>
+                            <input type="text" id="lname" name="lname" required><br><br>
+                        </div>
+                    </div>
+
+                    <label for="faddress-1">Address *</label><br>
```

Comment:
what was the reason for prefixing the form element ids with `f`? In general, class names, ids etc work best when they are named with meaningful names that a new developer can understand if they read your code for the first time.